editor/editor_node.cpp

```diff
@@ -6741,15 +6730,11 @@ EditorNode::EditorNode() {
 	write_movie_button->set_pressed(false);
 	write_movie_button->set_icon(gui_base->get_theme_icon(SNAME("MainMovieWrite"), SNAME("EditorIcons")));
 	write_movie_button->set_focus_mode(Control::FOCUS_NONE);
-	write_movie_button->connect("pressed", callable_mp(this, &EditorNode::_menu_option), make_binds(RUN_WRITE_MOVIE));
 	write_movie_button->set_tooltip(TTR("Enable Movie Maker mode.\nThe project will run at stable FPS and the visual and audio output will be recorded to a video file."));
-	// Restore these values to something more useful so it ignores the theme
-	write_movie_button->add_theme_color_override("icon_normal_color", Color(1, 1, 1, 0.4));
-	write_movie_button->add_theme_color_override("icon_pressed_color", Color(1, 1, 1, 1));
-	write_movie_button->add_theme_color_override("icon_hover_color", Color(1.2, 1.2, 1.2, 0.4));
-	write_movie_button->add_theme_color_override("icon_hover_pressed_color", Color(1.2, 1.2, 1.2, 1));
-	write_movie_button->add_theme_color_override("icon_focus_color", Color(1, 1, 1, 1));
-	write_movie_button->add_theme_color_override("icon_disabled_color", Color(1, 1, 1, 0.4));
+	// This button behaves differently, so color it as such.
+	write_movie_button->add_theme_color_override("icon_normal_color", Color(1, 1, 1, 0.7));
+	write_movie_button->add_theme_color_override("icon_pressed_color", Color(0.93, 0.33, 0.33));
```

Comment:
I recommend reusing the editor error color here:

```suggestion
	write_movie_button->add_theme_color_override("icon_pressed_color", get_theme_color(SNAME("error_color"), SNAME("Editor")));
```

The name is not very semantic for this use case, but it should be more visible on light theme (as the color is automatically adjusted in that case).

Reply:
Done.